lit/Driver/TestRepl.test

```diff
@@ -0,0 +1,7 @@
+# RUN: echo ':quit' | %lldb -x --repl -O 'expr 42' -S %S/Inputs/Print2.in -o 'expr 999999' -s %s 2>&1 | FileCheck %s
```

Comment:
Nitpick: you can use `-b` to get LLDB to exit after it's processed all the -O/-o/-S/-s options.